site2/docs/security-tls-transport.md

```diff
@@ -256,3 +259,4 @@ var client = PulsarClient.Builder()
                          .VerifyCertificateName(false)     //Default is 'false'
                          .Build();
 ```
+> Note that `VerifyCertificateName` refers to the configuration of hostname verification in the C# client.
```

Comment:
@blankensteiner Hi, could you help verify this?

Reply:
Hi @RobertIndie 
I can verify that by default it does not verify the certificate name.

Reply:
@blankensteiner Is `VerifyCertificateName` the same as the hostname verification in other language clients? It looks like they are all checking the `CommonName` in the certificate.

Reply:
@RobertIndie I have no idea. We register a [callback](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.remotecertificatevalidationcallback?view=net-6.0) when creating the [SslStream](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslstream.-ctor?view=net-6.0#system-net-security-sslstream-ctor(system-io-stream-system-boolean-system-net-security-remotecertificatevalidationcallback-system-net-security-localcertificateselectioncallback)) and then ignore the RemoteCertificateNameMismatch flag on the [SslPolicyErrors](https://docs.microsoft.com/en-us/dotnet/api/system.net.security.sslpolicyerrors?view=net-6.0) if VerifyCertificateName is set to false.